new-dti-website/components/slideshow.tsx

```diff
@@ -31,7 +31,7 @@ const ImageHeader: React.FC<{ imageName: string }> = ({ imageName }) => (
 const Slideshow: React.FC<SlideshowProps> = ({ selectedImage }) => {
   const image = imageNames[selectedImage ?? 0];
   return (
-    <div className="bg-white w-fit md:rounded-[20px] xs:rounded-lg relative z-20">
```

Comment:
Just wanted to address this since you're making changes! Do you think it might be better for the fonts to create a global css for them or extend them within the tailwind config! I believe it's creating hydration issues since the font seems to be rehashing on each upload when we're dynamically adding in the class font. 

{`ml-auto font-medium md:text-[14px] xs:text-[7px] ${ibm_plex_mono.className}`}>

I'm not too sure if we just want to just suppress it or address it? 

What do you think @cchrischen @andrew032011 ??

Reply:
Just double checking, is this likely the reason why when I click around the site locally and go to different pages, I sometimes get "Hydration failed"? I've noticed that when developing but have ignored it thus far.

Reply:
Yup! I guess if it hasn't really been giving anyone too many errors we can just use the suppressHydrationWarning like this: 
<time datetime="2016-10-25" suppressHydrationWarning />
In order to ignore the warning. I think it's mainly due to the dynamic font, which is a little weird because we set is as a constant.

Reply:
I'm not as familiar with the consequences of leaving it in. But my instinct says if there's an easy way to prevent it, I'd prefer that over supressing it

Reply:
Also double checking, you're referring to the parts of the code that have `${ibm_plex_mono. className}`? Not this line specifically right?

Reply:
Yes! not this line specifically but the `${ibm_plex_mono. className}`. I think consequences are minor but definitely something we can address when we add to the global css file or the config.